app/controllers/projects/samples/file_imports_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Projects
+  module Samples
+    # Controller actions for Project Samples File Import Controller
+    class FileImportsController < Projects::ApplicationController
+      include SampleFileImportActions
```

Comment:
Remove the `if Flipper.enabled?(:batch_sample_file_import)` from `config/routes/project.rb` and only include SampleFileImportActions `if Flipper.enabled?(:batch_sample_file_import)`